### PR TITLE
Added a tool to create an assembly cache based on the packages we ship and zip the contents for distribution

### PR DESCRIPTION
**Pull request by javiercn**
/cc @pranavkm

### CODE REVIEW

File: src/DependenciesPackager/Package-Dependencies.ps1 [hunk 1]
```diff
@@ -0,0 +1,42 @@
+Param(
```
Comment (pranavkm) [inline src/DependenciesPackager/Package-Dependencies.ps1]: How does this get invoked?

Comment (javiercn) [inline src/DependenciesPackager/Package-Dependencies.ps1]: .\Package-Dependencies.ps1 -ProjectJson C:\work\Test\split\ship\project.json -SourceFolders C:\work\Test\split\ship, C:\work\Test\split\ext, C:\work\Test\split\tmp -FallbackFeeds "http://www.nuget.org/api/v2" -PackagesFolder C:\work\Test\split -PackagesVersion "1.0.0" -ZipPath C:\work\Test\split"

Comment (javiercn) [inline src/DependenciesPackager/Package-Dependencies.ps1]: You can also do help Package-Dependencies

**Comment by Eilon** [comment]
I don't read PowerShell too good, but this seems a'ight.

File: src/DependenciesPackager/Package-Dependencies.ps1 (hunk 1 above)
Comment (pranavkm) [inline src/DependenciesPackager/Package-Dependencies.ps1]: Sorry I meant who's running the script? Build step in CI?

Comment (Eilon) [inline src/DependenciesPackager/Package-Dependencies.ps1]: Yes, basically the last step of Coherence-Signed.

**Comment by pranavkm** [comment]
:shipit:

**Pull request merged**